test/src/components/Button-test.js

```diff
@@ -26,4 +26,19 @@ describe('Button', () => {
     React.addons.TestUtils.Simulate.click(node);
     customHandleClick.called.should.be.true;
   });
+  it('should take in a color attribute', () => {
+    let coloredButton = render(<Button color={'teal'}>Submit</Button>).findTag('button');
+    coloredButton.props.className.should.contain('teal');
+    coloredButton.getDOMNode().getAttribute('class').should.contain('teal');
+  });
+  it('should take in boolean attributes', () => {
+    let disabledButton = render(<Button disabled>Submit</Button>).findTag('button');
+    disabledButton.props.className.should.contain('disabled');
+    disabledButton.getDOMNode().getAttribute('class').should.contain('disabled');
+  });
```

Comment:
This test checks if the boolean values are getting added to the classes properly